src/Components/DisplayAllArticles.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from "react";
+import axios from "axios";
+import ArticleTemplate from "./ArticleTemplate";
+
+class DisplayAllArticles extends Component {
+  constructor() {
+    super();
+    this.state = {
+      allArticles: []
+    };
+  }
+
+  componentDidMount() {
+    this.getAllArticles();
+  }
+
+  async getAllArticles() {
+    axios.get("http://localhost:3000/api/v0/articles")
```

Comment:
If you make this function àsync`then please use `await`. You are mixing styles.

Reply:
Fixed.